discovery-provider/src/models/models.py

```diff
@@ -581,6 +581,17 @@ def __repr__(self):
 play_item_id={self.play_item_id},\
 count={self.count}>"
 
+class LatestIndexingStates(Base):
+    __tablename__ = "latest_indexing_states"
+
+    tablename = Column(String, primary_key=True, nullable=False, index=False)
+    latest_indexing_state = Column(Integer, nullable=False, index=False)
```

Comment:
i think this class/model should be a bit clearer tbh - it looks like latest_indexing_state is the highest slot in this case? is this intended for generalization?

Reply:
yeah the intention was we might store blocknumbers or even in this case change to play primary key ids instead. ray suggested re-naming to indexing_checkpoint. does that sound good to you?